src/Sheet.php

```diff
@@ -265,7 +265,12 @@ protected function appendRow($row, $sheetExport)
             $row = $row->toArray();
         }
 
-        $this->append([$row], null, $this->hasStrictNullComparison($sheetExport));
+        if (is_array($row[0])) {
```

Comment:
You should add an isset check before the is_array.  This can given exceptions when someone wouldn't return an array with 0 index.